### DISCUSSION TITLE
对象存储可以添加minio么？

### DISCUSSION THREAD

**Discussion by hb-0** [AlistGo/alist#4920] [Q&A] [answered]
Minio兼容S3协议，理论上可以。但是我试了几次都提示签名不对。可以的话，请给一个配置示例。

**Comment by SeanHeuc** [comment]
最好提供下日志

**Reply by hb-0** [reply]
配置如下：
存储桶:http://<域名>:9000/test
Endpoint:http://<域名>:9000

页面提示：failed get objs: failed to list objs: SignatureDoesNotMatch: The request signature we calculated does not match the signature you provided. Check your key and signing method. status code: 403, request id: 17780DB7A4430616, host id: dd9025bab4ad464b049177c95eb6ebf374d3b3fd1af9251148b658df7ac2e3e8
alist是Docker部署的，docker logs -f alist看没有任何日志。

----
配置改为如下后：
存储桶:test
Endpoint:http://<域名>:9000

页面提示：
failed get objs: failed to list objs: RequestError: send request failed caused by: Get "http://test.<域名>:9000/?delimiter=%2F&marker=&prefix=": dial tcp: lookup test.<域名> on *******:53: no such host

**Reply by xhofe** [reply, marked as answer]
打开 `强制路径样式`

**Comment by anwen-anyi** [comment]
基本上主要的是这五个选项，因为没有`Minio`账号无法提供添加示例
- 存储桶 
- Endpoint 
- 地区 
- 访问密钥 Id
- 安全访问密钥
